docs/vectorizer/README.md

```diff
@@ -0,0 +1,673 @@
+# Automate AI embedding with pgai Vectorizer
+
+Vector embeddings transform text and documents into compact, semantically rich representations that enable context-aware searches beyond traditional keyword matching.
+
+While PostgreSQL excels at storing and querying these embeddings, synchronizing embeddings with source data has traditionally required custom solutions from developers.
+
+pgai Vectorizer provides a SQL-level interface that automates the embedding process within your database. By treating embeddings as a declarative feature—similar to an index—we simplify the entire workflow.
```

Comment:
don't love "SQL-level" as it's a python lib...

Reply:
I still feel we are continuously talking about different things when using the word Vectorizer. This comes from https://github.com/timescale/pgai/pull/695/files#diff-918dd3e75368bf16064b77ffdb8a33ebfee6d8fcf268f076732e4b77fcf6a4ebL14-L25. 

We sometimes refer to the Vectorizer as a whole solution which includes pgai library + Vectorizer worker. Sometimes we just refer to the actual config model found in `ai.vectorizer`.